### PR TITLE
Update including-sub-components.md

### LINKED ISSUE #43: Document rendering components via config
Add docs for frctl/fractal#573

### PR DESCRIPTION
**Pull request by justafish**
Adds documentation for the full rendered sub-components feature added in https://github.com/frctl/fractal/pull/573 and https://github.com/frctl/fractal/pull/646

This needs a release first though… :grin:

Fixes #43

### CODE REVIEW

File: docs/guide/components/including-sub-components.md [hunk 1]
```diff
@@ -225,3 +225,46 @@ In this case, when rendered the components will look like:
...
+  - name: active
+    context:
+      attributes: "class=\"is-active\""
+```
+
+```twig
+<!-- menu-local-tasks.twig -->
+{% if primary %}
+  <h2 class="visually-hidden">{{ 'Primary tabs'|t }}</h2>
+  <ul>{{ primary }}</ul>
```
Comment (mihkeleidast) [inline docs/guide/components/including-sub-components.md:256]: does this actually work? `primary` is an array...

File: docs/guide/components/including-sub-components.md [hunk 2]
```diff
@@ -225,3 +225,46 @@ In this case, when rendered the components will look like:
...
+{% endif %}
+{% if secondary %}
```
Comment (mihkeleidast) [inline docs/guide/components/including-sub-components.md:258]: is this necessary in the example? the primary example is enough in my opinion, also secondary is not defined in the config.

File: docs/guide/components/including-sub-components.md [hunk 3]
```diff
@@ -225,3 +225,46 @@ In this case, when rendered the components will look like:
...
+
+variants:
+  - name: active
+    context:
+      attributes: "class=\"is-active\""
+```
+
+```twig
+<!-- menu-local-tasks.twig -->
+{% if primary %}
```
Comment (mihkeleidast) [inline docs/guide/components/including-sub-components.md:254]: lose the if - it's not necessary to demonstrate the functionality.

**Review by mihkeleidast — CHANGES_REQUESTED** [review]

File: docs/guide/components/including-sub-components.md (hunk 1 above)
Comment (justafish) [reply docs/guide/components/including-sub-components.md:256]: it does! I'll update it to use an explicit loop so it's clearer though

**Comment by justafish** [comment]
@mihkeleidast thank you for the review, I've updated the PR per your feedback

**Review by mihkeleidast — APPROVED** [review]

File: docs/guide/components/including-sub-components.md [hunk 4]
```diff
@@ -225,3 +225,43 @@ In this case, when rendered the components will look like:
 You can see that in the `@search-box` component the button text is now the same as the text defined in the `button.config.yml` file.
 
 See the documentation on [referencing other component\'s data](../core-concepts/context-data.html#data-references) for more information on using this technique.
+
+### Fully rendering a sub-component
```
Comment (mihkeleidast) [inline docs/guide/components/including-sub-components.md:229]: can we add a "added in v1.3.0" tag here like we've used in [other places](https://fractal.build/guide/collections/configuration-reference.html#root)?

**Review by mihkeleidast — CHANGES_REQUESTED** [review]

**Comment by mihkeleidast** [comment]
I think this is fine to merge as soon as we get that tag added :) Since the feature was released a while ago, there's no need to wait for a release - although releasing the recent fix would certainly help. I'm working on it, as I'd like to have some other fixes in a release as well.

File: docs/guide/components/including-sub-components.md (hunk 4 above)
Comment (justafish) [reply docs/guide/components/including-sub-components.md]: @mihkeleidast done!

**Review by mihkeleidast — APPROVED** [review]

**Pull request merged**